### DISCUSSION TITLE
@formily/antd 添加 antd@5.x支持

### DISCUSSION THREAD

**Discussion by hulk-yin** [alibaba/formily#3594] [Ideas]
不支持antd@5.x，建议更新antd5.x支持及相关升级文档（如有构建要求，提供webpack、vite等构建升级说明）

https://codesandbox.io/s/peaceful-silence-w4cpbb?file=/index.tsx

**Comment by karlsbeard** [comment]
> 不支持[antd@5.x](mailto:antd@5.x)，建议更新antd5.x支持及相关升级文档（如有构建要求，提供webpack、vite等构建升级说明）
> 
> [codesandbox.io/s/peaceful-silence-w4cpbb?file=/index.tsx](https://codesandbox.io/s/peaceful-silence-w4cpbb?file=/index.tsx)

好像可以使用@formily/antd-v5来进行支持，这样可以直接用antd v5版本
